runtime/startup_aux.c

```diff
@@ -196,3 +186,124 @@ CAMLexport void caml_shutdown(void)
 
   shutdown_happened = 1;
 }
+
+void caml_init_exe_name(const char_os* exe_name)
+{
+  params.exe_name = exe_name;
+}
+
+void caml_init_section_table(const char* section_table,
+                             asize_t section_table_size)
+{
+  params.section_table = section_table;
+  params.section_table_size = section_table_size;
+}
+
+#ifndef NATIVE_CODE
```

Comment:
Why was this bytecode-specific code moved from `statup_byt.c` to `startup_aux.c`?

Reply:
I'm not keen on this either. I suspect the reasoning is that `caml_param` is immutable.
I've fixed this up with [ocaml-multicore#836](https://github.com/ocaml-multicore/ocaml-multicore/pull/836) which also highlighted a merge issue with `-config`

Reply:
Fixed in 19f2791